### DISCUSSION TITLE
gltf-transform in Docker

### DISCUSSION THREAD

**Discussion by mmlngl** [donmccurdy/glTF-Transform#941] [Show and tell]
Small thing, but I last night published gltf-transform as a docker image!

Repo:
https://github.com/mmmoli/3d-tools/tree/main/packages/gltf-transform

Docker image:
https://hub.docker.com/repository/docker/mmmoli/gltf-transform/general

Any suggestions are welcome!

Thank you @donmccurdy

**Comment by donmccurdy** [comment]
Looks great @mmmoli! 

No idea how easy this would be, but if you're applying KTX2 compression you may need to preinstall [KTX Software](https://github.com/KhronosGroup/KTX-Software) as well.

**Reply by mmlngl** [reply]
Great shout. I've made an issue to track the work: https://github.com/mmmoli/3d-tools/issues/1
